### ISSUE TITLE
height of lv_roller is abnormal

### ISSUE THREAD

**Issue by wachsh302** [kisvegabor/lvgl#6061]
### LVGL version

v8.3.11

### What happened?

use lv_roller with font size 20 of SourceHanSansCN_Normal: normal
but use lv_roller with font size 28 of SourceHanSansCN_Normal: the height is too small, text is truncated
![微信图片_20240411105541](https://github.com/lvgl/lvgl/assets/24709123/e5731b5c-c8d4-485a-a822-4e204ebe4ded)

**Comment by liamHowatt** [comment]
Hey,

I tried using that font at size 20 and 28 on a roller in v8.3. I used default theme and mono theme.

If I understand correctly, is the problem the top edge of the "00" is getting cut off? If so, try adding `lv_roller_set_visible_row_count(roller, 3)` to the roller. This is something that happens for all larger fonts.

Before
![image](https://github.com/lvgl/lvgl/assets/30486941/9469df24-de32-4e26-9286-3e1a6c4d4a4d)

After
![image](https://github.com/lvgl/lvgl/assets/30486941/32227553-bf11-4784-9234-4cedaa5a985c)


Or is the problem that the gray selection field is shifted upwards for the "30" roller? I could not reproduce that.
